Application/EdFi.Ods.Features/Controllers/AggregateDependencyController.cs

```diff
@@ -47,12 +48,13 @@ public IActionResult Get()
             }
 
             try
-            {       
-                    return Request.Headers["Accept"].ToString().EqualsIgnoreCase(CustomMediaContentTypes.GraphML)
-                    ? Ok(CreateGraphML(_resourceLoadGraphFactory.CreateResourceLoadGraph()))
-                    : Ok(GetGroupedLoadOrder(_resourceLoadGraphFactory.CreateResourceLoadGraph()));
+            {
+                return Request.GetTypedHeaders().Accept != null
+                    && Request.GetTypedHeaders().Accept.FirstOrDefault().MediaType.Value.EqualsIgnoreCase(CustomMediaContentTypes.GraphML)
```

Comment:
Should we be using FirstOrDefault or Any?

Reply:
Replaced with Any